conda_docker/conda.py

```diff
@@ -647,11 +685,21 @@ def build_docker_environment(
     with timer(LOGGER, "writing docker file"):
         image.write_filename(output_filename)
 
+def copy_pip_packages(target_dir, pip_files):
+    pip_targ_dir = os.path.join(target_dir, "opt/", "conda/")
```

Comment:
```suggestion
    pip_target_dir = os.path.join(target_dir, "opt", "conda")
```

Although I feel a little funny hardcoding `/opt/conda` in there. I wonder if we could use `default_prefix` similar to the `chroot_install` method